### PR TITLE
ticdc: update ga related docs

### PR DESCRIPTION
**Pull request by amyangfei**
### What is changed, added or deleted? (Required)

Add some new features in TiCDC GA version

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL @overvenus @leoppro @TomShawn

**Review by leoppro — COMMENTED** [review]
LGTM

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -586,3 +685,17 @@ sync-ddl = true
 5. 如果想在线 DDL，需要确保以下两点：
     1. 多个集群的 TiCDC 构成一个单向 DDL 同步链，不能成环，例如示例中只有 C 集群的 TiCDC 关闭了 `sync-ddl`。
     2. DDL 必须在单向 DDL 同步链的开始集群上执行，例如示例中的 A 集群。
+
+## 输出行变更历史值特性
+
+> **警告：**
+>
+> 目前环形同步属于实验特性，尚未经过完备的测试，请在生产环境中谨慎使用该特性。
```
Comment (overvenus) [inline ticdc/manage-ticdc.md:693]:
```suggestion
> 目前输出行变更历史值属于实验特性，尚未经过完备的测试，请在生产环境中谨慎使用该特性。
```

**Review by overvenus — COMMENTED** [review]
Rest LGTM

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -586,3 +685,17 @@ sync-ddl = true
 5. 如果想在线 DDL，需要确保以下两点：
     1. 多个集群的 TiCDC 构成一个单向 DDL 同步链，不能成环，例如示例中只有 C 集群的 TiCDC 关闭了 `sync-ddl`。
     2. DDL 必须在单向 DDL 同步链的开始集群上执行，例如示例中的 A 集群。
+
+## 输出行变更历史值特性
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:689]:
```suggestion
## 输出行变更的历史值 <span class="version-mark">New in v4.0.5</span>
```

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -586,3 +685,17 @@ sync-ddl = true
...
+
+> **警告：**
+>
+> 目前环形同步属于实验特性，尚未经过完备的测试，请在生产环境中谨慎使用该特性。
+
+在默认配置下同步任务输出的 open protocol 行变更数据只会包含变更后的值，不包含变更前行的值，这种情况不支持 TiDB 4.0 [新的 Collation 框架](/character-set-and-collation.md#新框架下的排序规则支持)，也不满足 open protocol 的消费端使用行变更历史值的需求。从 v4.0.5 开始，TiCDC 支持通过开关开启输出行变更历史值的特性。开启改特性，需要在 changefeed 的配置文件根级别指定以下配置，开启后 open protocol 的输出格式参考 [TiCDC 开放数据协议 - Row Changed Event](/ticdc/ticdc-open-protocol.md#row-changed-event)，使用 MySQL sink 时 TiDB 4.0 新的 Collation 特性也会自动支持。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:695]: open protocol 是 TiCDC 开放数据协议吗？
```suggestion
在默认配置下同步任务输出的 open protocol 行变更数据只包含变更后的值，不包含变更前行的值，因此该输出数据不支持 TiDB 4.0 [新的 Collation 框架](/character-set-and-collation.md#新框架下的排序规则支持)，也不满足 open protocol 的消费端使用行变更历史值的需求。

从 v4.0.5 开始，TiCDC 支持输出行变更数据的历史值。若要开启该特性，需要在 changefeed 的配置文件的根级别指定以下配置。开启后，open protocol 的输出格式参考 [TiCDC 开放数据协议 - Row Changed Event](/ticdc/ticdc-open-protocol.md#row-changed-event)，使用 MySQL sink 时 TiDB 4.0 新的 Collation 特性也会自动支持。
```

Comment (amyangfei) [reply ticdc/manage-ticdc.md:695]: 是的

File: ticdc/manage-ticdc.md (hunk 1 above)
Comment (leoppro) [reply ticdc/manage-ticdc.md:693]: 直接写不能用于生产环境吧

File: ticdc/manage-ticdc.md [hunk 4]
```diff
@@ -586,3 +685,19 @@ sync-ddl = true
 5. 如果想在线 DDL，需要确保以下两点：
     1. 多个集群的 TiCDC 构成一个单向 DDL 同步链，不能成环，例如示例中只有 C 集群的 TiCDC 关闭了 `sync-ddl`。
     2. DDL 必须在单向 DDL 同步链的开始集群上执行，例如示例中的 A 集群。
+
+## 输出行变更的历史值 <span class="version-mark">New in v4.0.5</span>
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:689]:
```suggestion
## 输出行变更的历史值 <span class="version-mark">从 v4.0.5 版本开始引入</span>
```

File: ticdc/manage-ticdc.md [hunk 5]
```diff
@@ -293,6 +364,34 @@ cdc cli changefeed remove --pd=http://10.0.10.25:2379 --changefeed-id simple-rep
 
 - `--changefeed=uuid` 为需要操作的 `changefeed` ID。
```
[the remark this answers is not in the document]
Comment (amyangfei) [reply ticdc/manage-ticdc.md:365]: 修正了，需要用 --changefeed-id

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**